webview.h

```diff
@@ -900,6 +905,9 @@ class win32_edge_engine {
       m_window = CreateWindowW(L"webview", L"", WS_OVERLAPPEDWINDOW,
                                CW_USEDEFAULT, CW_USEDEFAULT, 640, 480, nullptr,
```

Comment:
At some point I think we should do something about the magic numbers `640` and `480` here.